### DISCUSSION TITLE
VIP歌曲还是不能听

### DISCUSSION THREAD

**Discussion by painter686** [qier222/YesPlayMusic#1847] [Q&A]
网易云的会员歌曲还是不能听，而且歌单放到这首歌时会显示这首歌的封面，但却播放着另一首歌，歌单里VIP试听的歌，在本应用里打开直接显示无法播放。

**Comment by GrassBlock1** [comment]
看起来应该是 UnblockNeteaseMusic 的问题，请尝试
1. 本地部署 UnblockNeteaseMusic 并在代理选项中配置。
2. 更换音源、关闭获取Flac后清理缓存重试，具体参考相关文档
3. ~~充个会员~~

**Comment by painter686** [comment]
邮件已收到！
